base/custom_metrics.go

```diff
@@ -251,10 +252,6 @@ func (t *customMetricsTask) run(exp *Experiment) error {
 		return err
 	}
 
-	if exp.driver == nil {
```

Comment:
I think this was left over after `collect-metrics-database` task was restructured to `custom_metrics`. We do not use `exp.driver` here anymore.